.github/rundecks/GISS_ONLY.R

```diff
@@ -232,8 +232,7 @@ aer_rad_forc=0   ! turn off aerosol radiative forcing diagnostics
 cloud_rad_forc=1 ! turn on cloud radiative forcing diagnostics
 
 ! diagnostics
-! SUBDD='OH:4 NO:4 O3:4 NO2:4 CO:4 CH4:4 PS:4' ! TODO: Turn these on
-SUBDD=''
```

Comment:
We actually don't want these in the `GISS_ONLY` rundeck. The work currently underway on diagnostics is only for the coupled GISS-GC model (i.e., `GISS_GC_14` rundeck). So instead of enabling these here, I think we just want to leave `SUBDD=''` and put a more appropriate comment describing the situation, as opposed to the `TODO` note.